ansible/roles/routemgmt/tasks/clean.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Remove API Gateway route management actions.
+
+- name: remove route management actions
+  shell: ./uninstallRouteMgmt.sh {{ catalog_auth_key }} {{ groups['edge'] | first }} {{ catalog_namespace }} {{ cli_path }} chdir="{{ openwhisk_home }}/ansible/roles/routemgmt/scripts"
+  environment:
+    OPENWHISK_HOME: "{{ openwhisk_home }}"
+  ignore_errors: True
```

Comment:
the other openwhisk roles like invoker, controller, etc have to have 'ignore_errors: True` because they can't remove the docker containers (invoker, controller, etc) due to this issue: https://github.com/openwhisk/openwhisk/issues/199
This only happen in an AIO topology, in a distributed topo it does not happen cause invoker containers are not deployed in the same machine as controller.

So, the router clean step should be able to clean the entities, not ignore the errors.